### ISSUE TITLE
Don't advertise using the master branch as the version

### ISSUE THREAD

**Issue by ponychicken** [hallee/eslint-action#8]
Especially for CI it is a good choice to use a controlled environment. We copied the code from the README file, which contains 
````
uses: hallee/eslint-action@master
````
I think it would be better to recomente using a fixed version:
````
uses: hallee/eslint-action@1.0
````

**Comment by darshkpatel** [comment]
+1 using a tested release version is a better idea than relying on a changing master

**Comment by jeffbyrnes** [comment]
As it turns out, the current `HEAD` of the master branch is the only pointer that contains any of this repo’s real improvements over the original it forked (`gimenete/eslint-action`).

But it definitely is not quite working in some cases.

**Comment by hallee** [comment]
Fixed as of #13, thanks. I'll try to not make breaking changes going forward
